### PR TITLE
Add pod and container name to log context

### PR DESCRIPTION
**Pull request by SupriyaKasten**
## Change Overview

Add pod name and container name to context in addition to actionset name and phase name 

## Pull request type

Please check the type of change your PR introduces:
- [ ] Work in Progress
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Trivial/Minor
- [ ] Bugfix
- [x] Feature
- [ ] Documentation

### CODE REVIEW

File: pkg/function/restore_data.go [hunk 1]
```diff
@@ -176,6 +177,10 @@ func (*restoreDataFunc) Exec(ctx context.Context, tp param.TemplateParams, args
 	}
 	// Validate and get optional arguments
 	restorePath, encryptionKey, pod, vols, backupTag, backupID, err := validateAndGetOptArgs(args)
+
+	if pod != "" {
```
Comment (tdmanv) [inline pkg/function/restore_data.go]: doesn't this Function also use pod runner?

File: pkg/function/restore_data_all.go [hunk 2]
```diff
@@ -128,6 +129,7 @@ func (*restoreDataAllFunc) Exec(ctx context.Context, tp param.TemplateParams, ar
 	output := make(map[string]interface{})
 	for _, pod := range pods {
 		go func(pod string) {
+			ctx = field.Context(ctx, field.PodNameKey, pod)
```
Comment (tdmanv) [inline pkg/function/restore_data_all.go]: same here

File: pkg/function/restore_data.go (hunk 1 above)
Comment (SupriyaKasten) [reply pkg/function/restore_data.go]: It does, but the pod name set here is different than the one that is generated by pod runner.

Comment (tdmanv) [reply pkg/function/restore_data.go]: I would either drop this or rename the key since it's getting overwritten anyway

Comment (julio-lopez) [reply pkg/function/restore_data.go]: Use a different key in order to tell apart the 2 pod names?

**Pull request merged**